ca/certificate-authority_test.go

```diff
@@ -333,12 +334,17 @@ var FarPast = time.Date(1950, 1, 1, 0, 0, 0, 0, time.UTC)
 
 var log = mocks.UseMockLog()
 
+var exPA = cmd.PAConfig{
```

Comment:
This name is super confusing. paConf is better.

But this is probably mooted once we pass in a real PA into the constructors that need it.